src/diffusers/models/unet_1d.py

```diff
@@ -54,16 +75,20 @@ def __init__(
         out_channels: int = 2,
         extra_in_channels: int = 0,
         time_embedding_type: str = "fourier",
-        freq_shift: int = 0,
```

Comment:
This we cannot remove anymore sadly as we have dance diffusion now which relies on this config parameter https://huggingface.co/harmonai/maestro-150k/blob/main/unet/config.json#L35 -> could we leave the name as is here please :-) Otherwise we'd break dance diffusion :sweat_smile:

Reply:
If you feel super strongly about it we could also adapt the config of dance diffusion online and leave as is